### PR TITLE
feat: play all API and CLI

### LINKED ISSUE #149: feat: play all (& glob) API and config
We should expose a `playAll` method.

Also we should support a config option to play a capture, glob, or all by default on server start.

As a follow up, `play` could support accepting a glob pattern so that `playAll` is alias of `play('*')`.

### PR DESCRIPTION
**Pull request by AndersDJohnson**
fixes #149

### CODE REVIEW

File: packages/mockyeah/app/index.js [hunk 1]
```diff
@@ -71,6 +72,7 @@ module.exports = function App(config) {
   app.record = recorder(app);
   app.recordStop = recordStopper(app);
   app.play = player(app);
+  app.playAll = playAller(app);
```
Comment (ryanricard) [inline packages/mockyeah/app/index.js]: Baller.

File: packages/mockyeah/app/playAller.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    const dirs = files.filter(file =>
+      // eslint-disable-next-line no-sync
+      fs.statSync(path.join(capturesDir, file)).isDirectory()
+    );
+
+    // if (!captureNames.length) {
+    //   console.log(chalk.red('No captures available to start'));
+    //   console.log(chalk.red('Record one by running: mockyeah record [name]'));
+    //   throw new Error('');
+    // }
```
Comment (ryanricard) [inline packages/mockyeah/app/playAller.js]: What's the plan here?

Comment (AndersDJohnson) [reply packages/mockyeah/app/playAller.js]: @ryanricard I wasn't sure if we should show an error in the console when there are no captures found when playing all. For now, I've just decided to not warn, so I've just removed this commented code.

File: packages/mockyeah/app/index.js (hunk 1 above)
Comment (AndersDJohnson) [reply packages/mockyeah/app/index.js]: I know, the naming sucks...

**Pull request merged**